content/networking/ip-optimization-strategies/index.md

```diff
@@ -0,0 +1,95 @@
+
+The [Amazon VPC CNI](../vpc-cni/) plugin assigns each pod an IP address from the VPC CIDR(s), which can consume a substantial number of IP addresses from your VPCs depending on your workload.
+
+Therefore, when designing your AWS networking architecture, it is important to optimize EKS IP consumption at the VPC and at the node level. This will help you mitigate IP exhaustion issues and increase the pod density per node.
+
+In this section, we will discuss techniques that can help you achieve these goals.
+
+
+## Optimize node-level IP consumption
+
+[Prefix delegation](https://docs.aws.amazon.com/eks/latest/userguide/cni-increase-ip-addresses.html) is a feature of Amazon Virtual Private Cloud (Amazon VPC) that allows you to assign IPv4 or IPv6 prefixes to your Amazon Elastic Compute Cloud (Amazon EC2) instances. This can improve your efficiency, reduce your costs, and give you more flexibility. Prefix delegation can improve your efficiency in a number of ways by increasing the amount of available IP addresses for your Amazon EC2 nodes. This means that you can fit more pods on each instance, which can improve your pod density and IP utilization. Prefix delegation is flexible and can be used with both IPv6 and custom networking. Prefix delegation, can also help you to achieve a better pod density and workload allocation on nodes. 
+
+Check these sections of the EKS Best Practices for more information:
+
+* [Prefix Delegation with Linux nodes](../prefix-mode/index_linux/),
+
+* [Prefix Delegation with Windows nodes](../prefix-mode/index_windows/). 
+
+## Mitigate IP exhaustion
+
+### Use IPv6 (recommended)
+
+Adopting IPv6 is the easiest way to work around RFC1918 limitations; we strongly recommend to consider adopting IPv6 as your first option when choosing network architectures. IPv6 provides a significantly larger total IP address space, and Kubernetes cluster administrators can focus on migrating and scaling applications without devoting effort towards working around IPv4 limits.
+
+Amazon EKS clusters support both IPv4 and IPv6. By default, EKS clusters use IPv4 address space. Specifying IPv6 based address space at cluster creation time will enable the use of IPv6 clusters. In an IPv6 EKS cluster, pods and services will receive IPv6 addresses while **maintaining the ability for legacy IPv4 endpoints to connect to services running on IPv6 clusters and viceversa**. All the pod-to-pod communication within a cluster is always IPv6. Within a VPC (/56), the IPv6 CIDR block size for IPv6 subnets is fixed at /64. This provides 2^64 (approximately 18 quintillion) IPv6 addresses allowing to scale your deployments on EKS. Please, check in the EKS Best Practices [section dedicated to IPv6](../ipv6/) and the [Amazon EKS Section of the IPv6 workshop](https://catalog.workshops.aws/ipv6-on-aws/en-US/lab-6) for more details. 
```

Comment:
Last sentence : 

For detailed information please see the [Running IPv6 EKS Clusters](https://aws.github.io/aws-eks-best-practices/networking/ipv6/) section and for hands-on experience please see the [Understanding IPv6 on Amazon EKS](https://catalog.workshops.aws/ipv6-on-aws/en-US/lab-6) section of the [Get hands-on with IPv6](https://catalog.workshops.aws/ipv6-on-aws/en-US) workshop.